roles/install-app/templates/kernelci-backend.cfg

```diff
@@ -1,5 +1,6 @@
 master_key = '{{ master_key }}'
-storage_url = 'http://storage.kernelci.org/'
+storage_url = 'https://storage.kernelci.org/'
```

Comment:
Are you sure about the trailing `/` here?  Should it not be `https://storage.kernelci.org` instead?

Reply:
Also the commit summary "use https" is a bit vague.

Reply:
Links currently work with the the trailing '/' there so I wouldn't recommend to remove it without checking for possible breakage.